actions/check_team_permissions_for_repository.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: check_team_permissions_for_repository
+runner_type: python-script
+pack: github
+description: >
+  Check team permissions for a repository.
+  Example:
+    st2 run github.check_team_permissions_for_repository  organization="organization" owner="owner" repo="reponame" team_slug="team_id" api_user="token_name"
```

Comment:
Example uses parameter organization, but yaml states parameter is called org - so these don't match.